Blockzilla/SupportUtils.swift

```diff
@@ -4,17 +4,75 @@
 
 import Foundation
 
+public enum SupportTopic: String {
+    case focusHelp = "focus"
+    case klarHelp = "klar"
+    case usageData = "usage-data"
+
+    /// These are the additional supported languages, next to the default language the document is written in. At
+    /// some later stage SUMO will do an automatic redirect based on the Language-Accepts header. For now we will
+    /// have to maintain a static list of supported languages.
+
+    var supportedLanguages: Set<String> {
+        switch self {
+        case .focusHelp:
+            return Set(["fr", "zh-tw", "hi-in", "es", "ru", "jp", "it", "pt", "id", "pl"])
```

Comment:
I would really put these in alphabetical order, if nothing else to avoid checking the entire list every single time you need to modify or check it.

```
var supportedLanguages: Set<String> {
    switch self {
    case .focusHelp:
        return Set(["es", "fr", "hi-in", "id", "it", "jp", "pl", "pt", "ru", "zh-tw"])
    case .klarHelp:
        return Set()
    case .usageData:
        return Set(["de", "es", "fr", "hi-in", "id", "it", "jp", "pl", "pt", "ru", "zh-tw"])
    }
}
```